lib/build.js

```diff
@@ -86,33 +85,6 @@ class BuildModel extends BaseModel {
     }
 
     /**
-     * Lazy load a user model
-     * @property user
-     * @return {Promise}    Resolves to the user associated with this build
-     */
-    get user() {
```

Comment:
We don't use this function any where, and we don't need to set `username` and `scmContext` in constructor if the function would be removed.